tests/ci/sync_pr.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+"""Script for automatic sync PRs handling in private repos"""
+
+import sys
+
+from get_robot_token import get_best_robot_token
+from pr_info import PRInfo
+from github_helper import GitHub
+
+
+def main():
+    gh = GitHub(get_best_robot_token())
+
+    pr_info = PRInfo()
+    assert pr_info.merged_pr, "BUG. merged PR number could not been determined"
+
+    prs = gh.get_pulls_from_search(
+        query=f"type:pr [Sync] ClickHouse/ClickHouse#{pr_info.merged_pr}",
+        repo="ClickHouse/clickhouse-private",
+    )
+    if len(prs) > 1:
+        print(f"WARNING: More than one PR found [{prs}]")
+    elif len(prs) == 0:
+        print("WARNING: No Sync PR found")
+        sys.exit(0)
+
+    pr = prs[0]
+
+    if pr.state == "closed":
```

Comment:
These checks for states will go away since the query will cover them.

Reply:
I added this for better visibility

Reply:
There should be another filter in the query `state:open`, so we won't get more than one PR